.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-**Prep Release: (assigned to:)**
-_Due: <3-business-days-before-release>_
-- [ ] Merge all features and fixes into the `dev` branch
-- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
-- [ ] Update dependencies and increment extension version in the `dev` branch.
-- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
-- [ ] Close out or punt remaining GitHub issues for the current milestone
+**Prep DTFx Release: (assigned to:)**
+_Due: <2-3-business-days-before-release>_
+- [ ] Increment updated DTFx packages (either DT-AzureStorage only or DT-AzureStorage, DT-Core, and DT-Redis)
+- [ ] Delete DTFx test packages from myget.
+- [ ] Update OneBranch durabletask repo, run signing pipeline (using these notes:  [Durable Task Framework Build Pipeline] (https://microsoft.sharepoint.com/teams/AzureWebjobs/_layouts/OneNote.aspx?id=%2Fteams%2FAzureWebjobs%2FSiteAssets%2FAzureWebjobs%20Notebook&wd=target%28Planning%2FFunctions%20Post-GA.one%7CA43CF112-7272-481A-B23E-9AA5CA8EEE06%2FDurable%20Task%20Framework%20Build%20Pipeline%7CD0946823-6FB0-44E3-A57F-E252617B69CD%2F%29
+onenote:https://microsoft.sharepoint.com/teams/AzureWebjobs/SiteAssets/AzureWebjobs%20Notebook/Planning/Functions%20Post-GA.one#Durable%20Task%20Framework%20Build%20Pipeline&section-id={A43CF112-7272-481A-B23E-9AA5CA8EEE06}&page-id={D0946823-6FB0-44E3-A57F-E252617B69CD}&end) ).
+- [ ] Publish DTFx packages to myget for testing. (either DT-AzureStorage only or DT-AzureStorage, DT-Core, DT-Redis, and DT-Emulator)
 
-
-**Validation (assigned to: )**
+**Prep Release (assigned to: )**
 _Due: <2-business-days-before-release>_
-- [ ] Run private performance tests and ensure no regressions
-- [ ] Smoke test Functions V1 and Functions V3 .NET apps
-- [ ] Smoke test JavaScript and Python apps
-- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release
+- [ ] Update Durable Functions references (Analyzer? DTFx?) and increment version.
+- [ ] Add Durable Functions package to myget staging feed.
+- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release.
```

Comment:
We should probably provide an expected package size to the notes

Reply:
People can just check the previous package size.

Reply:
Our current package size is under ~470KB. I'm not necessarily opposed to setting some arbitrary limit (say 500 or 600KB) for now so that we make sure that we have some consistent definition of "suprisingly heavier".